modules/web-console/frontend/app/components/form-field/components/form-field-size/controller.ts

```diff
@@ -76,7 +76,16 @@ export default class PCFormFieldSizeController<T> implements IInputErrorNotifier
         if (!this.min) this.min = 0;
         if (!this.sizesMenu) this.setDefaultSizeType();
         this.$element.addClass('form-field');
-        this.ngModel.$render = () => this.assignValue(this.ngModel.$viewValue);
+        this.ngModel.$render = () => {
```

Comment:
Can this be covered by tests?

Reply:
Yes.

Reply:
Updated test "Switch editor measure" in modules/web-console/frontend/app/components/form-field/components/form-field-size/index.spec.ts to check initial measure detection.